conifer/model.py

```diff
@@ -30,8 +30,8 @@ def compile(self):
     def decision_function(self, X, trees=False):
         return self.backend.decision_function(X, self.config, trees=trees)
 
-    def build(self):
-        self.backend.build(self.config)
+    def build(self, reset=False, csim=False, synth=True, cosim=False, export=False):
```

Comment:
After some more testing, I realised this breaks the VHDL backend. The fix is fairly straightforward, and I can add it afterwards if you like. 
Basically, `reset=False, csim=False, synth=True, cosim=False, export=False` here needs to change to `**kwargs`, and that needs to propagate to a few other places.

Reply:
I don't have much experience using kwargs so I think I will let you fix this afterwards